src/Hooks/Dispatchers/ReplaceCommandHookDispatcher.php

```diff
@@ -17,6 +18,15 @@ class ReplaceCommandHookDispatcher extends HookDispatcher
 {
 
     /**
+     * @var LoggerInterface|null
+     */
+    protected $logger = null;
+
+    public function setLogger(LoggerInterface $logger) {
```

Comment:
I had assumed that ReplaceCommandHookDispatcher wasn't a container service and consequently would not receive the Logger via inflection.